relay-event-normalization/src/eap/mod.rs

```diff
@@ -387,7 +387,11 @@ pub fn normalize_db_attributes(annotated_attributes: &mut Annotated<Attributes>)
 
     if let Some(attributes) = annotated_attributes.value_mut() {
         if let Some(normalized_db_query) = normalized_db_query {
+            let mut normalized_db_query_hash = format!("{:?}", md5::compute(&normalized_db_query));
+            normalized_db_query_hash.truncate(16);
```

Comment:
Does this hash need to be compatible with something that already exists? Rephrased, can we choose a different hash here?